chrome/content/zoteroquicklook.js

```diff
@@ -453,7 +453,9 @@ Checks the attachment file or writes a content of a note to a file and then push
 	*/
 
 	openQuickLook: async function(items) {
-
+		this.viewerBaseArguments=['-p'];
+        this.viewerExecutable = Zotero.File.pathToFile("/usr/bin/qlmanage");
+		
```

Comment:
This might work for macOS, but it would most likely break ZQL for Linux/Windows.

Reply:
Should add the flag to avoid errors on Windows.
```
             if (!Zotero.isWin) {
                 Zotero.debug("ZoteroQuickLook: Error");
                 return;
             }
```

Reply:
@qqobb Yes, this is only tested on macOS. I have already reverted the change in `.js` file.